Libraries/Connect/Implementation/Streaming/URLSessionStream.swift

```diff
@@ -15,11 +15,13 @@
 import Foundation
 
 /// Stream implementation that wraps a `URLSession` stream.
-final class URLSessionStream: NSObject {
-    private var closedByServer = false
+final class URLSessionStream: NSObject, @unchecked Sendable {
```

Comment:
The comments below about `@unchecked` are useful; however, it may make it even more clear to put a comment more adjacent to the `@unchecked Sendable` up here. I actually was 🤔 "why is this unchecked? isn't it an NSObject with a bunch of `let` properties?" prior to looking a bit further down.

Reply:
Updated!